events/events.go

```diff
@@ -0,0 +1,251 @@
+package events
+
+import (
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"github.com/SAP/service-fabrik-cli-plugin/errors"
+	"github.com/SAP/service-fabrik-cli-plugin/helper"
+	"github.com/SAP/service-fabrik-cli-plugin/constants"
+	"github.com/cloudfoundry/cli/plugin"
+	"github.com/fatih/color"
+	"github.com/olekukonko/tablewriter"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+)
+
+var (
+	httpClientDisabledSecurityCheck *http.Client
+	httpClientEnabledSecurityCheck  *http.Client
+)
+
+type EventCommand struct {
+	cliConnection plugin.CliConnection
+}
+
+func Initialize() {
+	httpClientDisabledSecurityCheck = CreateHttpClient(true)
+	httpClientEnabledSecurityCheck = CreateHttpClient(false)
+}
+
+func NewEventsCommand(cliConnection plugin.CliConnection) *EventCommand {
+	command := new(EventCommand)
+	command.cliConnection = cliConnection
+	return command
+}
+
+func AddColor(text string, textColor color.Attribute) string {
+	printer := color.New(textColor).Add(color.Bold).SprintFunc()
+	return printer(text)
+}
+
+type Configuration struct {
+	ServiceBroker       string
+	ServiceBrokerExtUrl string
+	SkipSslFlag         bool
+}
+
+func GetBrokerName() string {
+	return GetConfiguration().ServiceBroker
+}
+
+func GetExtUrl() string {
+	return GetConfiguration().ServiceBrokerExtUrl
+}
+
+func GetskipSslFlag() bool {
+	return GetConfiguration().SkipSslFlag
+}
```

Comment:
As also asked in previous review [here](https://github.com/cloudfoundry-incubator/service-fabrik-cli-plugin/pull/2#discussion_r198434239), where are functions GetBrokerName , GetExtUrl and GetskipSslFlag used? Remove if not.